log-output/pkg/index/index.go

```diff
@@ -1,21 +1,26 @@
 package index
 
-import "time"
+import (
+	"fmt"
+	"time"
+)
 
 const MaxIndexLogs = 100
 
 type Index struct {
-	Name     string
+	Name     string    `json:"name"`
 	FirstLog time.Time `json:"first_log"`
 	LastLog  time.Time `json:"last_log"`
 	LogCount int64     `json:"log_count"`
 	Logs     [][]byte  `json:"-"`
+	Path     string    `json:"path"`
```

Comment:
What does this adding this get us? it is not a full url so clients will need to do formatting on their end anyway.